Lib/doctest.py

```diff
@@ -1532,59 +1545,61 @@ def summarize(self, verbose=None):
         notests = []
         passed = []
         failed = []
-        totalt = totalf = 0
-        for x in self._name2ft.items():
-            name, (f, t) = x
-            assert f <= t
-            totalt += t
-            totalf += f
-            if t == 0:
+        total_tries = total_failures = total_skipped = 0
+        for item in self._stats.items():
+            name, (failures, tries, skipped) = item
```

Comment:
```suggestion
        for name, (failures, tries, skipped) in self._stats.items():
```

Reply:
*item* is used below, I prefer to keep it: ``failed.append(item)``.

Reply:
Ah, I missed that, apologies